lib/matobjplist.gi

```diff
@@ -929,19 +928,8 @@ InstallMethod( \*, "for two plist matrices",
     if not IsIdenticalObj(a![BDPOS],b![BDPOS]) then
         ErrorNoReturn("\\*: Matrices not over same base domain");
     fi;
-    l := ListWithIdenticalEntries(Length(a![ROWSPOS]),0);
-    for i in [1..Length(l)] do
-        if b![RLPOS] = 0 then
-            l[i] := b![EMPOS];
-        else
-            v := a![ROWSPOS][i];
-            w := ZeroVector(b![RLPOS],b![EMPOS]);
-            for j in [1..a![RLPOS]] do
-                AddRowVector(w,b![ROWSPOS][j],v[j]);
-            od;
-            l[i] := w;
-        fi;
-    od;
+    l := a![ROWSPOS]*b![ROWSPOS];
+    #l := PROD_LIST_SCL_DEFAULT(a![ROWSPOS],b![ROWSPOS]);
```

Comment:
Here we use the optimized matrix multiplication functions from the kernel, which is part of the reason for the performance improvements.